modules/apps/item-selector/item-selector-taglib/src/test/java/com/liferay/item/selector/taglib/internal/util/GroupItemSelectorProviderRegistryUtilTest.java

```diff
@@ -46,6 +50,31 @@ public class GroupItemSelectorProviderRegistryUtilTest {
 	public static final LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		BundleContext bundleContext = SystemBundleUtil.getBundleContext();
```

Comment:
@MrJohn1911 Please make this `bundleContext` a static field so that you don't need to get it repeatedly in the test methods.